platform/cavium : Download debian packages

garyachy opened the pull request.

marian-pritsak commented on platform/cavium/cavm-sai.mk:
```diff
@@ -1,12 +1,20 @@
+$(XPSHELL)_URL = $(CAVM_SAI_GITHUB)/$(XPSHELL)
+SONIC_ONLINE_DEBS += $(CAVM_LIBSAI) $(CAVM_SAI) $(XP_TOOLS) $(XPSHELL)
+
+#$(CAVM_LIBSAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
```
Why still keep those lines?

garyachy replied:
We need this for debug purpose

marian-pritsak replied:
You can just copy files to target/debs, and make won't try to download them even if they are in SONIC_ONLINE_DEBS group.
Using SONIC_COPY_DEBS is an unnecessary step here.

marian-pritsak commented on platform/cavium/cavm-sai.mk:
```diff
@@ -1,12 +1,20 @@
 # Cavium SAI
 
+CAVM_SAI_GITHUB = https://github.com/garyachy/OpenXPS/raw/sai_0.9.5/SAI/cavm-sai/
```
Are these files going to be updated within same branch?
If yes, use commit hash in URL instead of branch name.

garyachy replied:
We always are going to use latest and greatest.

marian-pritsak replied:
If, for instance, SAI headers are changed, you won't be able to compile older version of SONiC.
But it's up to you.

garyachy replied:
Cavium provides both SAI headers and SAI adapter library.

In case if SONIC is upgraded to use SAI 1.0 headers, Cavium build will be broken until we change both our SAI headers and SAI adapter library.

Anyway, thanks for your review comments.

lguohan replied:
btw, I think sonic uses sai 0.9.4 branch

garyachy commented:
@lguohan I have addressed your comment. Please merge.
Also please advise who can create project and job for Cavium on Jenkins?
I believe we are ready for Jenkins integration  as soon as this PR is merged.

lguohan commented:
@garyachy , job is already created. https://sonic-jenkins.westus.cloudapp.azure.com/job/cavium/job/buildimage-cavm-all/. I'll merge this PR once once other builds pass.

garyachy commented:
@lguohan where I will find build results?
